modular/stonekeep/code/jobs/job_types/adventurer/types/combat/amazon.dm

```diff
@@ -13,8 +13,9 @@
 	..()
 	H.mind?.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
 	H.mind?.adjust_skillrank(/datum/skill/combat/bows, 3, TRUE)
+	H.mind?.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)//the role had a roll for swords but no skill for it
```

Comment:
Leave swords at 3 so that getting a sword isn't shafting yourself. Either that or, preferrably, assign these skill levels on the choices below, like granting sword and bow levels only if you picked sword, and spear and crossbow levels if you chose spear (I believe the blowgun takes crossbow skills?)